### ISSUE TITLE
Tuning for the Data Augmentation CLI parameters

### ISSUE THREAD

**Issue by tttthomasssss** [RasaHQ/rasa#7920]
This concerns some of the command line arguments that can be passed to the NLU data augmentation CLI tool (incoming with PR #7584 ).

There are two parameters that have not been empirically validated:
* `--num-intents` which is set to `5` by default and which is the number of intents to pick for data augmentation among the 5 intents with lowest F1-Score, lowest Recall, lowest Precision, least amount of training data, and the 5 intents most frequently confused.
* `--paraphrase-score-threshold` which is set to `0.8` and is used for the data augmentation strategy based on maximum vocabulary expansion to ensure that any generated paraphrases are still "similar" enough to the original data point and avoid the influx of completely unrelated utterances (as they would maximise the criterion of expanding the vocabulary by the largest amount).

**Comment by Ghostvv** [comment]
@tttthomasssss are you working on this issue? Should it be moved to `in progress`?

**Comment by tttthomasssss** [comment]
No, not yet - its part of the todos @dakshvar22 and I have decided to put on the backburner until the main PR is merged.
